ckanext/canada/schemas/presets.yaml

```diff
@@ -4075,3 +4075,25 @@ presets:
     display_snippet: multiple_choice.html
     validators: scheming_multiple_choice
     output_validators: scheming_multiple_choice_output
+
+# Field = Schema.
+- preset_name: canada_resource_validation_schema
+  values:
+    field_name: schema
+    form_snippet: null
+    form_panel: _exclude_from_form
+    required: false
+    display_snippet: null
+    validators: canada_resource_schema_validator
```

Comment:
we could use `validators: ignore` and a custom `output_validators` that returns the value we want instead. That way we don't need to update every resource in the db, all we need to do is reindex the datasets (the fixed value we choose will always be output by the output_validators)

Or we could hack the validation extension to only use our options and not require any special resource fields..

Reply:
@wardi it seems that using `validators: ignore` and an output validator does not work? Does there need to be a value for the output validator to fire off??

Reply:
@wardi I have added in a `canada_blank_value_creator` which just sets the validation options and schema fields to an empty value so that the output validators work.

I just do not really wish to modify the validation fork/extension more than we have to.